newapi/ooniapi/utils.py

```diff
@@ -22,3 +23,23 @@ def nocachejson(*a, **kw):
     resp.cache_control.max_age = 0
     resp.cache_control.no_cache = True
     return resp
+
+
+def jer(error_id, tpl, *a, **kw) -> Response:
+    """Error handler with localization support
+    Return error_id, error_msg human friendly msg and errdata dict
+    Set Cache-Control: No-Cache
+    Set HTTP error code from the first optional argument or default to 400
+    """
+    # The function definition is parsed by ../scan_error_handlers.py
+    if error_id == "":
+        error_id = tpl.title().replace(" ", "")
+    if a:
+        http_code = a[0]
+    else:
+        http_code = 400
```

Comment:
I found the previous implementation with `code=CODE` more readable than this implementation where we pass a list and then we magically know that the error code is the first element. I see why we cannot revert easily from the previous implementation because we're passing the error data as `kw`. But still it's a bit "meh".